### ISSUE TITLE
Add Test/Train split; modified code so that model can be applied to new data; show some results; add some links 

### ISSUE THREAD

**Issue by gitclem** [snorkel-team/snorkel-tutorials#248]
(Sorry, I'm a noob with doing pull requests...)

I made some changes to improve getting_started.ipynb

Attached are my changes (I had to rename `getting_started.ipynb` to `getting_started.ipynb.txt` in order to attach.)

I tried to make a branch and check in my changes and make a [pull request](https://opensource.com/article/19/7/create-pull-request-github) but for some reason it didn't work.

Changes made:
1. added some internal navigation links
2. split CountVectorizer.fit_transform() into two steps so that the resulting vectorizer transform function can be applied to new data but use the previous fit on the train data
3. added a split of the data into train and test set
4. output some example entries with spam probabilities from both the train and test set.

[getting_started.ipynb.txt](https://github.com/snorkel-team/snorkel-tutorials/files/6601876/getting_started.ipynb.txt)

My pull request steps:
```
git checkout -b improved_getting_started
git remote add upstream https://github.com/snorkel-team/snorkel-tutorials
git checkout -b improved_getting_started
(edited getting_started.ipynb)
git add getting_started.ipynb
git commit -m "improvements to getting_started.ipynb; added code showing how model can be applied to new data; added internal links; show some actual probabilities"
git push -u origin improved_getting_started
remote: Permission to snorkel-team/snorkel-tutorials.git denied to gitclem.
fatal: unable to access 'https://github.com/snorkel-team/snorkel-tutorials.git/': The requested URL returned error: 403
```
